rosette/src/protobuf/Ob.proto

```diff
@@ -89,32 +89,67 @@ message LetExpr {
     Object bindings = 1;
     Object body = 2;
 }
-message LetrecExpr {}
-message MethodExpr {}
-message Niv {}
-message NullExpr {}
-message Proc {}
-message ProcExpr {}
-message QuoteExpr {}
-message RblAtom {}
+message LetrecExpr {
+}
+message MethodExpr {
+    Object identity = 1;
+    Object formals = 2;
+    Object body = 3;
+}
+message Niv {
+}
+message NullExpr {
+}
+message Proc {
+    Object env = 1;
+    Object code = 2;
+    Object id = 3;
+    Object source = 4;
+}
+message ProcExpr {
+}
+message QuoteExpr {
+    Object expr = 1;
+}
+message RblAtom {
+}
 message RblBool {
     bool value = 1;
 }
 message RBLstring {
     string value = 1;
 }
-message ReflectiveMethodExpr {}
-message RequestExpr {}
-message SendExpr {}
-message SeqExpr {}
-message SetExpr {}
-message StdExtension {}
-message StdMthd {}
+message ReflectiveMethodExpr {
+}
+message RequestExpr {
+    Object target = 1;
+    Object msg = 2;
+}
+message SendExpr {
+}
+message SeqExpr {
+    Object subexprs = 1;
+}
+message SetExpr {
+    Object trgt = 1;
```

Comment:
Please be consistent with naming.  Don't use `trgt` here and `target` on 125 unless the Rosette code does it this way.

Reply:
I've used exclusively the names that exist in the Rosette objects.  If we'd like to standardize on a set of names for Roscala, that would be fine with me.

Reply:
OK, since it's the Rosette code that does this, let's leave it the same.